packages/db/.expo/README.md

```
@@ -0,0 +1,13 @@
+> Why do I have a folder named ".expo" in my project?
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)